ibis/bigquery/tests/conftest.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+import pandas as pd
+import ibis.util
+from ibis.bigquery import client as ibc  # noqa: E402
+
+
+pytest.importorskip('google.cloud.bigquery')
+pytestmark = pytest.mark.bigquery
+
+
+PROJECT_ID = 'bq-for-ibis'
+DATASET_ID = 'ci_{}'.format(ibis.util.guid())
+TABLE_ID = 'table_{}'.format(ibis.util.guid())
+
+
+@pytest.fixture(scope='session')
+def df():
+    return pd.DataFrame(dict(
+        int_column=[1], float_column=[2.],
+        string_column=['a'], bool_column=[True],
+    ))
+
+
+@pytest.fixture(scope='session')
+def client():
+    return ibc.BigQueryClient(PROJECT_ID, DATASET_ID)
+
+
+@pytest.fixture(scope='session')
+def client_with_table(client, df):
+    schema = ibc.infer_schema_from_df(df)
+    tuples = list(tuple(v.tolist()) for (k, v) in df.iterrows())
+    #
+    bq_dataset = client._proxy.get_dataset(client.dataset_id)
+    bq_dataset.create()
+    bq_dataset.reload()
+    bq_table = bq_dataset.table(TABLE_ID, schema)
+    bq_table.create()
+    bq_table.reload()
+    bq_table.insert_data(tuples)
```

Comment:
You'll have more reliable tests if you upload from a file and wait for the load job to complete. Example:

https://github.com/GoogleCloudPlatform/google-cloud-python/blob/6345c06ceb151d07761abb36f577909a3b76d7d4/docs/bigquery/snippets.py#L401-L408

Note: line-delimited JSON or Avro rather than CSV will be more robust in general, but CSV is probably fine for a test fixture like this.